main/medium/src/reaper.rs

```diff
@@ -1695,6 +1696,116 @@ impl<UsageScope> Reaper<UsageScope> {
         PositionInQuarterNotes::new(qn)
     }
 
+    /// Returns the MIDI tick (ppq) position corresponding to the start of the measure.
+    ///
+    /// # Safety
+    ///
+    /// REAPER can crash if you pass an invalid take.
+    pub unsafe fn midi_get_ppq_pos_start_of_measure(
+        &self,
+        mut take: MediaItemTake,
+        ppq: PositionInPPQ,
+    ) -> PositionInPPQ
+    where
+        UsageScope: AnyThread,
+    {
+        let ppq = self.low.MIDI_GetPPQPos_StartOfMeasure(take.as_mut(), ppq.0);
```

Comment:
```suggestion
        let ppq = self.low.MIDI_GetPPQPos_StartOfMeasure(take.as_ptr(), ppq.0);
```

Doesn't require `mut`.